### ISSUE TITLE
[Bug]: Task Output Folder View produces way too many results

### ISSUE THREAD

**Issue by alessandroperilli** [Comfy-Org/ComfyUI_frontend#720]
### Frontend Version

1.2.44

(but this issue existed since the feature was introduced in previous versions)

### Expected Behavior

For debugging purposes, for each image generation, [AP Workflow](https://perilli.com/ai/comfyui/) produces a few intermediate images throughout the pipeline. For example: if the control panel is configured to enable the face detailer function and the upscaler, APW will generate:

- IMG1 of the basic txt2img operation
- IMG2 of the face detailing operation applied to IMG1
- IMG3 of the upscaling operation applied to IMG3

Throughout the pipeline, for debugging purposes, APW also prints a few lines of text. So, the previous example would produce 2-3 text outputs in console detailing the parameters used for the generation.

According to this, and as I understand the new Task Output Folder View feature, I'd expect to see a folder output with 3 to 5-6 items in it (depending if only images are counted, or if also text is counted - this is unclear to me). 

### Actual Behavior

Actual behavior with all functions but txt2img enabled:

<img width="292" alt="Screenshot 2024-09-03 at 08 41 32" src="https://github.com/user-attachments/assets/423fb62c-7097-4b24-9a80-c79ad865a62c">

And yes. Clicking on the task folder number really shows 1212 items:

<img width="358" alt="Screenshot 2024-09-03 at 08 52 02" src="https://github.com/user-attachments/assets/76dce2eb-de25-46e4-b720-0a131bb4aae3">

### Steps to Reproduce

I see this with APW 11.0, which is still in Early Access. Didn't try with APW 10.0 which is GA. 
Happy to provide a copy if you want to investigate.

### Debug Logs

```powershell
Nothing out of the ordinary in the terminal logs.
```

### Browser Logs

Attached.
[192.168.1.6-1725350263197.log](https://github.com/user-attachments/files/16845348/192.168.1.6-1725350263197.log)

### What browsers do you use to access the UI ?

Other

### Other

Vivaldi on macOS

**Comment by huchenlei** [comment]
Will it be helpful if we group the outputs by type? Saved outputs first then temp results (Previews, etc)

**Comment by alessandroperilli** [comment]
Yes, but it would be even more helpful if I could filter out the things I don't want to see. For example, I am REALLY happy that the queue system now captures all the intermediate images APW generates. I want that. But I don't care anything about the text outputs (because I can already see them in the terminal). 

Regardless: I'm pretty sure APW doesn't generate 1212 outputs for each run. So, either I don't understand what's being captured, or there's some problem.
